pkg/smartcontract/zkpbinding/binding.go

```diff
@@ -60,7 +60,9 @@ const (
 	// verification contract is circuit-specific, i.e. corresponds to a specific
 	// single constraint system. Thus, every new circuit requires vew verification
 	// contract to be generated and deployed to the chain.
-	goVerificationTmpl = `// Package main contains verification smart contract that uses Neo BLS12-381
+	goVerificationTmpl = `//Code generated by neo-go zkpbinding.GenerateVerifier; DO NOT EDIT.
+//
```

Comment:
> but not be attached to the package clause

So it has to be a separate comment, right?

Reply:
Does it mean it needs space between two comments?

Reply:
I think so. Try finding some similar tools and check how exactly it's structured there.

Like
```
// Code generated by "stringer -type=Role"; DO NOT EDIT.
```
But this doesn't have package-level documentation, unfortunately. My primary concern is correct package doc. You can check it with `godoc` though.